tools/doprofile.py

```diff
@@ -270,10 +270,14 @@ def setXAxisSteps(self):
                     step = step * 365/12
 
                 profile["l"] = []
-                if useNetcdfTime and profile["layer"].source().startswith("NETCDF:"):
+                if useNetcdfTime and (profile["layer"].source().startswith("NETCDF:") or
+                                      profile["layer"].source().endswith(".nc")):
                     try:
                         import netCDF4
```

Comment:
You could also use `scipy.io.netcdf` for fallback which might work okay if the time units are very easy to parse like `seconds since {some reference date}`. But it is reasonable to expect users working with netCDF to install `netCDF4`.